### PR TITLE
update-collections-privacy-switch

### PR DESCRIPTION
**Pull request by JatinKumar001**
Hey, I fix the collection privacy switch.

Issue no. #10128

### CODE REVIEW

File: wagtail/admin/templates/wagtailadmin/collections/_privacy_switch.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
     <div class="privacy-indicator {% if is_public %}public{% else %}private{% endif %}">
         {% trans "Privacy" %}
-        <button type="button" data-a11y-dialog-show="set-privacy" class="status-tag primary">
+        <button type="button" data-a11y-dialog-show="set-privacy" class="button button-small button-secondary primary">
```
Comment (lb-) [inline wagtail/admin/templates/wagtailadmin/collections/_privacy_switch.html:14]:
```suggestion
        <button type="button" data-a11y-dialog-show="set-privacy" class="button button-small button-secondary">
```

I think the `primary` should also be removed

Comment (JatinKumar001) [reply wagtail/admin/templates/wagtailadmin/collections/_privacy_switch.html:14]: ok sure

Comment (JatinKumar001) [reply wagtail/admin/templates/wagtailadmin/collections/_privacy_switch.html]: Hey, I remove it and updated the branch.

**Review by lb- — APPROVED** [review]
Looking good now - thanks @JatinKumar001 
I will rebase, add changelog and push up to your branch.

I have added you to the contributor list as (Jatin Kumar) - let me know if you want a different name listed.

**Before**
Screenshot 2023-02-25 at 7 39 02 am

**After**

Screenshot 2023-02-25 at 7 41 02 am

**Pull request merged**